src/wwElement.vue

```diff
@@ -140,10 +134,35 @@ export default {
                 ...initialValue.filter(selection => !this.options.map(option => option.value).includes(selection))
             );
 
-            // await to avoid mismatch (multiselect not rendering custom tags)
-            await this.$nextTick();
+            // We set internalValue after the options to avoid mismatch
+            this.internalValue = initialValue;
+        },
+        /**
+         * We need to avoid to have a value not present in options
+         * So here we take care of not removing an used option
+         */
+        refreshOptions() {
+            // we removed unused options
+            this.options = this.options.filter(option => this.internalValue.includes(option.value));
 
-            this.internalValue = [...initialValue];
+            // Then we add the new initial options and avoid duplicate
+            const initialOptions = Array.isArray(this.content.options) ? this.content.options : [];

```

Comment:
its dangerous here, tu as un filter juste après, mais juste in case, copie le tableau si tu le manipule ;)